cmd/analyzer/main.go

```diff
@@ -437,10 +444,17 @@ func DependencyOutput(ctx context.Context, providers map[string]provider.Interna
 				continue
 			}
 			for u, ds := range deps {
+				var outputDepDag []konveyor.DepDAGItem
+				// trim m2 repo prefix for java deps
+				if name == "java" {
+					outputDepDag = trimDagItemPrefixes(ds)
```

Comment:
sorry one small request...I think hardcoding for java provider here is unnecessary. there is no need to change prefix fields on dependencies as it isn't output to users in any incidents.

Reply:
I think this is for the `--dep-output-file` for the user. Just checking for the java provider here as the trim() function it's calling is only used to trim the local m2 repo prefix. Wanted to avoid issue with other dep providers. However, if I'm off-track, please let me know.

Reply:
I think it is fine to leave the prefixes as-is in the output @shawn-hurley wdyt?